prebid-mobile/code-integration-ios.md

```diff
@@ -93,6 +77,16 @@ PBBannerAdUnit *__nullable adUnit1 = [[PBBannerAdUnit alloc] initWithAdUnitIdent
     	   andPrimaryAdServer:PBPrimaryAdServerDFP];
 ```
 
+The above example is for app using DFP as the primary ad server. If you're using MoPub, please use the API like this:
```

Comment:
Suggest changing to: "The preceding example is for apps using DFP as the primary ad server. For apps using MoPub, modify step 3 to use PBPrimaryAdServerMoPub, as shown here:"